### PR TITLE
fix: ignore nil value in array convert

### PR DESCRIPTION
**Pull request by carlclone**

### CODE REVIEW

**Comment by carlclone** [comment]
#2590

File: pkg/cast/cast.go [hunk 1]
```diff
@@ -905,14 +913,22 @@ func ToUint64Slice(input interface{}, sn Strictness) ([]uint64, error) {
 	return result, nil
 }
 
-func ToFloat64Slice(input interface{}, sn Strictness) ([]float64, error) {
+func ToFloat64Slice(input interface{}, sn Strictness, anc ARRAY_NIL_CONVERT) ([]float64, error) {
```
Comment (ngjaying) [inline pkg/cast/cast.go:916]: Shall we make similar changes to other ToXXXSlice functions?

Comment (carlclone) [reply pkg/cast/cast.go]: Those methods don't seem to affect the aggregation function. Should I modified them ? maybe in a new pull request .

**Review by ngjaying — APPROVED** [review]
Thank you @carlclone

**Pull request merged**